File: Java/Laki2/H1/StringLengthComparator.java
/* TIEP5.1 Lausekielinen ohjelmointi II
Harjoitus 1, tehtävä 5
Eveliina Toivanen (eveliina.toivanen@tuni.fi)
*/

import java.util.Scanner;
public class StringLengthComparator {
    public static void main(String[] args) {
        
        
        // 
        Scanner lukija = new Scanner(System.in);
        
        System.out.println("Hello! I compare the lengths of two strings."); 
        System.out.println("Please, enter the first string:"); 
        String ekaRivi = lukija.nextLine();
        
        System.out.println("Please, enter the second string:"); 
        String tokaRivi = lukija.nextLine();
        
        int ekaRiviPituus = ekaRivi.length();
        int tokaRiviPituus = tokaRivi.length();
        
        if (ekaRiviPituus > tokaRiviPituus) {
            System.out.println("\"" + ekaRivi + "\"" + " is longer than " 
                               + "\"" + tokaRivi + "\"" + ".");
        }

        if (ekaRiviPituus == tokaRiviPituus) {
            System.out.println("\"" + ekaRivi + "\"" + " is as long as " 
                               + "\"" + tokaRivi + "\"" + ".");
        }

        if (ekaRiviPituus < tokaRiviPituus) {
            System.out.println("\"" + ekaRivi + "\"" + " is shorter than " 
                               + "\"" + tokaRivi + "\"" + ".");
        
        }
    }
}

File: Python/LAKI1/H4/string_length_comparator.py
# TIEP1.1 Lausekielinen ohjelmointi I
# Harjoitus 4, tehtävä 4
# Eveliina Toivanen (toivanen.eveliinarai@gmail.com)

# Ohjelma kysyy käyttäjältä kaksi merkkijonoa ja tarkastaa, ovatko ne yhtä pitkiä
# vai onko jompi kumpi jonoista pidempi ja kertoo tuloksen tulosteessa.

print('Hello! I compare the lengths of two strings.')
print('Please, enter the first string:')
merkkijono_1 = str(input())
print('Please, enter the second string:')
merkkijono_2 = str(input())

if len(merkkijono_1) < len(merkkijono_2):
    print('"', merkkijono_1, '" is shorter than "', merkkijono_2, '".', sep='')

if len(merkkijono_1) == len(merkkijono_2):
    print('"', merkkijono_1,'" is as long as "', merkkijono_2,'".', sep='')

if len(merkkijono_1) > len(merkkijono_2):
    print('"', merkkijono_1, '" is longer than "', merkkijono_2, '".', sep='')

